src/Check/Memcached.php

```diff
@@ -72,7 +72,7 @@ public function check()
                 || false === $stats[$authority]
             ) {
                 // Attempt a connection to make sure that the server is really down
-                if (! @$memcached->getLastDisconnectedServer($this->host, $this->port)) {
+                if ($memcached->getLastErrorCode() !== 0) {
```

Comment:
Why changed ?

Reply:
@sasezaki There was a error that getLastDisconnectedServer() need to have exact 0 argument, but has 2

Reply:
Hmm, this code path doesn't seem to be tested at all: we can't change this without an integration test :thinking:

Reply:
Ok, so I should go back to previous version and do some if for PHP version?

Reply:
@Ocramius $memcached->getLastDisconnectedServer return array if there was any error.I changed how it should be handled in here and removed my proposition with getLastErrorCode().